assignments/003-illumina-sequencing-questions/illumina-seq-homework.md

```diff
@@ -0,0 +1,59 @@
+
+
+These are the questions that appear in the video about Illumina sequencing by synthesis that
+can be viewed at: [https://eriqande.github.io/erics-captioned-vids/vids/illumina-sbs/](https://eriqande.github.io/erics-captioned-vids/vids/illumina-sbs/).
+
+Copy this template to a new file in the same directory called simply
+`illumina-seq-homework.md`.  Please answer each question by typing in the space below each question, or by retaining the words corresponding to
+your answers in the "dual-choice in parentheses" questions.
+
+When you are done.  Save the file then commit it
+to a branch called `illumina-seq` and push it back to your
+fork of the course repo on GitHub then send me a pull request.
+
+
+
+* Q1: True or False. This animation is showing that each of the two "fragments" is a double-stranded segment of DNA. (_from 30.0 to 33.0 seconds in the video_).
+
+False.
+
+* Q2: How does the sequence of the fragment on top compare to the sequence of the fragment on the bottom? (_from 36.5 to 39.0 seconds in the video_).
+
+The fragments are complementary strands.
+
+* Q3: What is the purpose of the sequencing binding sites? Your answer should include the words "oligonucleotide," "polymerase," "3'-hydroxyl," and "synthesize." (_from 39.0 to 40.5 seconds in the video_).
+
+The sequencing binding site is an inserted DNA sequence designed to bind downstream sequencing oligonucleotides (or primers). Primers are short single strands of synthetic RNA or DNA that contain a free 3'-hydroxyl group. DNA polymerase can only add nucleotides to the 3'-hydroxyl group of an existing sequence and synthesizes DNA in the 5' to 3' direction. Therefore the sequencing binding site is necessary to bind a sequencing primer and provide a platform for DNA polymerase to conduct 'sequencing by synthesis'. 
+
+* Q4: Given the direction of travel of the polymerase, is the Original Template anchored to the lawn by its 3' or its 5' end? (_from 78.0 to 81.0 seconds in the video_).
+
+The Original Template is anchored by its 3' end.
+
+* Q5: How does the remaining strand relate to the Original Template? Is the remaining strand anchored by its 3' or 5' end? (_from 85.0 to 88.0 seconds in the video_).
+
+The remaining strand is anchored by its 5' end.
+
+* Q6: Choose one from the parentheses: the purple part on the remaining strand is (the same as || complementary to) the purple lawn oligos.  (_from 88.2 to 92.2 seconds in the video_).
+
+Complementary to
```

Comment:
I think it is the same as the purple lawn oligos---it started off, itself, as a purple lawn oligo.